README.md

```diff
@@ -1,4 +1,4 @@
-Authors Haven - A Social platform for the creative at heart.
```

Comment:
@meshack-mbuvi any reason for removal of this title?

Reply:
Seen it. It should be there. Let me correct it.